src/containters/Home/components/HeroView/HeroView.js

```diff
@@ -4,15 +4,14 @@ import { IoIosArrowForward } from 'react-icons/io';
 import { useEffect, useState } from 'react';
 import { heroView } from '../../services/heroView';
 import { BrowserRouter as Router, Link } from 'react-router-dom';
-const HeroView = () => {
-    const [heroViewData, setHeroViewData] = useState({});
-
+const HeroView = (props) => {
+    // const [heroViewData, setHeroViewData] = useState({});
+    
     useEffect(() => {
         Promise.all([heroView()])
             .then(res => {
-                // console.log('heroView', res[0].data);
-                setHeroViewData(res[0].data);
-                console.log(heroViewData);
+                // props.setHeroViewData(res[0].data);
```

Comment:
- Please remove this useEffect entirely as it will run and make an extra API call which badly affect performance and has no need.